src/ui/profile_preferences.ui

```diff
@@ -42,7 +42,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>5</number>
```

Comment:
You don't want this change as it means the form doesn't open on the first t.. - oh, I think I put a workaround in the C++ code to avoid that.